lib/tasks/dev.rake

```diff
@@ -0,0 +1,43 @@
+namespace :dev do
+  require 'factory_bot_rails'
+  require 'faker'
+
+  desc 'Generate test data'
+  task :generate_data, [:dev_num] => :environment do |_, args|
+    set_args(args)
+
+    begin
+
+      res_pro_langs = create_programming_language
```

Comment:
@ntrung2006 
`res_pro_langs` this name does not make sense. Could you change to another name? ex: `programming_languages`...

Reply:
It mean 'res_pro_lang' => `result_programming_languages` after created. @thienhv @@

Reply:
@ntrung2006 t
`result_programming_languages` make sense than `res_pro_langs`

but I will use `programming_languages`

Reply:
Fixed!